scripts/intfutil

```diff
@@ -131,6 +132,19 @@ def appl_db_sub_intf_keys_get(appl_db, sub_intf_list, sub_intf_name):
     return appl_db_sub_intf_keys
 
 
+def appl_db_port_speed_parse(in_speed, optics_type):
+    """
+    Parse the speed received from application DB
+    """
+    # fetched speed is in megabits per second
+    speed = int(in_speed)
+    if optics_type == OPTICS_TYPE_RJ45:
```

Comment:
I think we should check the exact speed values to decide whether its Gbps or Mbps. CAT6 RJ45 cable can support 1Gbps speed as well, so the port type doesn't seem to be the right choice to decide G or M.

Reply:
updated following what we agreed, on RJ45 port, if speed <= 1000M, then display it with unit 'M', otherwise use 'G'

Reply:
@keboliu for "PortChannel1" the "Type" is "N/A", so why the speed is showing "1000M" in the output?